spec/object/directory_spec.rb

```diff
@@ -92,6 +92,7 @@
   end
 
   describe "#overwrite_current_version" do
+    # Create the object root directory
```

Comment:
My Rubocop-related PR was just merged, so now I think this comment could move here: https://github.com/sul-dlss/ocfl-rb/blob/main/spec/support/temp_directory.rb